src/modules/Elsa.Workflows.Api/Endpoints/WorkflowDefinitions/Execute/Endpoint.cs

```diff
@@ -20,6 +20,7 @@ public override void Configure()
     {
         Post("/workflow-definitions/{definitionId}/execute");
         ConfigurePermissions("exec:workflow-definitions");
+        AllowAnonymous();
```

Comment:
I think you added this for testing purposes. Let's remove it.
For testing these endpoints that require authenticated users, you can create an access token using the `/identity/login` endpoint using the following curl:

```shell
curl --location --request POST 'https://localhost:5001/elsa/api/identity/login' \
--header 'Content-Type: application/json' \
--data-raw '{
    "username": "admin",
    "password": "password"
}'
```

The result contains an access token you can include as a bearer token in the Authorization header. For example:

```shell
curl --location --request POST 'https://localhost:5001/elsa/api/workflow-definitions/94afd3fa039e4efa931affab952d831f/execute' \
--header 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoiQWRtaW4iLCJwZXJtaXNzaW9ucyI6IioiLCJuYmYiOjE2NzIwODg3MDYsImV4cCI6MTY3MjA5MjMwNiwiaWF0IjoxNjcyMDg4NzA2LCJpc3MiOiJodHRwOi8vZWxzYS5hcGkiLCJhdWQiOiJodHRwOi8vZWxzYS5hcGkifQ.6LBaF8HY_QF7sxlgj96IWmDzk_vPY9KcWgwMMxx8XMQ' \
--header 'Content-Type: application/json'
```